certbot/certbot/_internal/main.py

```diff
@@ -507,6 +507,13 @@ def _report_next_steps(config: interfaces.IConfig, installer_err: Optional[error
                 "Certificates created using --csr will not be renewed automatically by Certbot. "
                 "You will need to renew the certificate before it expires, by running the same "
                 "Certbot command again.")
+        elif _is_interactive_only_auth(config):
+            steps.append(
+                "This certificate will not be renewed automatically by Certbot. The --manual "
+                "plugin requires the use of an authentication hook script (--manual-auth-hook) "
+                "in order to support autorenewal. To renew this certificate, repeat this same "
```

Comment:
```suggestion
                "in order to support autorenewal, which this certificate invocation did not provide. To renew this certificate, repeat this same "
```

Reply:
I don't think I understand the suggestion. Is the point to explicitly state what was missing? 

To avoid introducing a new term ("certificate invocation"), what about:

>This certificate will not be renewed automatically. Autorenewal of --manual certificates requires the use of an authentication hook script (--manual-auth-hook) but one was not provided. To renew this certificate, repeat this same certbot command before the certificate's expiry date.

Reply:
The point is to make it clear that "the --manual plugin requires the use of an authentication hook script (--manual-auth-hook) in order to support autorenewal" does not apply to what the user just did. Like, say, "it requires it, and you did not provide it." Just to be super clear.

But yes, your wording gets that across and is much more clear!

Reply:
Okey dokey, text and screenshots updated.